frontend/src/components/parties/PartyProfile.js

```diff
@@ -63,6 +79,38 @@ export class PartyProfile extends Component {
     });
   }
 
+  openEditPartyModal = (event, party, onSubmit, title, isPerson, provinceOptions) => {
+    const address = party.address[0];
+    const initialValues = {
+      email: party.email && party.email !== "Unknown" ? party.email : null,
+      first_name: party.first_name ? party.first_name : null,
+      party_name: party.party_name ? party.party_name : null,
+      phone_ext: party.phone_ext ? party.phone_ext : null,
+      phone_no: party.phone_no ? party.phone_no : null,
+      address_line_1: address.address_line_1 ? address.address_line_1 : null,
+      address_line_2: address.address_line_2 ? address.address_line_2 : null,
+      city: address.city ? address.city : null,
+      post_code: address.post_code ? address.post_code : null,
+      sub_division_code: address.sub_division_code ? address.sub_division_code : null,
+      suite_no: address.suite_no ? address.suite_no : null,
```

Comment:
These values are coming from the API, so the JSON payload should have them set to `null` if they're not present (ie. already failing the truthy ternary checks here). Is there a reason why we need to use the ternary at all?

I get the impression that (for example) `suite_no` would be `null`. It would fail the ternary check, and the property would be set to `null`. The same thing would result from `suite_no: address.suite_no`.